pages/common/rsync.md

```diff
@@ -4,7 +4,7 @@
 > Does not allow transfer between two remote hosts
 > Can transfer single files or files matched by pattern
 
-- transfer file from local to remote host
+- transfer file from local to remote host, it supports regex.
```

Comment:
can you replace ", it supports regex" with "(Note: filename supports regex)"? The current way doesn't form a proper sentence.